site/content/components/Hero.jsx

```diff
@@ -1,26 +1,26 @@
 export default function Hero() {
   return (
-    <main className="lg:relative">
-      <div className="mx-auto w-full max-w-7xl pt-16 pb-20 text-center lg:pt-48 lg:pb-32 lg:text-left">
+    <section className="lg:relative">
+      <div className="font-archivo mx-auto w-full max-w-7xl pt-16 pb-20 text-center lg:pt-48 lg:pb-32 lg:text-left">
```

Comment:
good to be able to have a named font and can we default a font - so we don't have to specify exactly. ie. just change the standard serif or sans etc.

Reply:
generally i don't want users having to write to `font-archivo`